VERSION

```diff
@@ -0,0 +1 @@
+V0.0.16
```

Comment:
used when build helm related things.

```
# Get release version from VERSION file
RELEASE_VERSION = $(shell cat VERSION)
```

Reply:
So, this is only for the helm char version. Right? How about moving it into[helm/api-testing](https://github.com/LinuxSuRen/api-testing/tree/master/helm/api-testing) if I understand it correctly?

By the way, I prefer to use `v0.0.16` instead of `V0.0.16`. Normally the prefix `v` is lower-case.

Reply:
sgtm, I will move